docs/UserGuide.adoc

```diff
@@ -234,12 +242,86 @@ Format: `warn INDEX q/MIN_QUANTITY`
 
 Example:
 
-* `warn 1 q/100` +
+* `warn 5 q/100` +
 This sets the minimum quantity threshold for good at index 1 with an quantity of 100.
 
+.Demo for `warn` command
+image::UG/warn.png[width="790"]
+
 [NOTE]
 When a new good is added into the inventory, its minimum quantity threshold is set at 0.
 
+=== Locating transactions by search criteria:
+
+Display list of transactions that fulfills the given search criteria.
+
+3 types of search criteria:
+
+. transaction type
+. `Name` of `supplier`
+. `GoodName` of `Good`
+
+Format: `find-t [TRANSACTION TYPE] [n/NAME] [g/GOOD NAME]`;
+
+.GUI before demo
+image::UG/original.png[width="790"]
+
+Example:
+
+* Search by transaction type: +
+`find-t buy` +
+display all `buy` transactions.
+
+NOTE: Currently, there is only two types of transaction: `buy` and `sell`.
+
+.Demo for `find-t` command that uses only [TRANSACTION TYPE] criteria
+image::UG/find-t buy.png[width="790"]
+
+* Search by `Name` of `Supplier`: +
+`find-t n/alex bernice` +
+display all transactions that related to `Alex` or `Bernice`.
+
+****
+* The search is case insensitive. e.g `bernice` will match `Bernice`
+* The order of the keywords does not matter. e.g. `Bernice Yu` will match `Yu Bernice`
+* Only the name is searched.
+* Only full words will be matched e.g. `bernice` will not match `bernices`
+* Supplier matching at least one keyword will be returned (i.e. `OR` search). e.g. `Alex Bernice` will return `Alex Yeoh`, `Bernice Yu`
+****
+
+.Demo for `find-t` command that uses only [NAME] criteria
+image::UG/find-t name.png[width="790"]
+
+* Search by `Good Name` of `Good`: +
+`find-t g/apple noodle` +
+display all transactions that related to `Apple` or `Noodle`.
+
+.Demo for `find-t` command that uses only [GOOD NAME] criteria
+image::UG/find-t good name.png[width="790"]
+
+`Combination of criteria` +
+Criteria can be combined to give a more constraint search.
+
+Example:
+
+* Search by transaction type and `GoodName` of `Good: +
+`find-t buy n/apple noodle` +
+display all `buy` transactions that is related to `Apple` or `Noodle`.
+
+.Demo for `find-t` command that uses [TRANSACTION TYPE] and [GOOD NAME] criteria.
+image::UG/combination.png[width="790"]
```

Comment:
Will this picture be redundant? We have shown all examples of the individual ones